website/payments/templates/payments/email/tpay_withdrawal_notice_mail.txt

```diff
@@ -0,0 +1,19 @@
+Dear {{ name }},
+
+In the past period, you have made use of Thalia Pay. With this email, we would like to inform you that in the upcoming days, we will withdraw an amount of € {{ total_amount|floatformat:2 }} from your bank account {{ bank_account.iban }} {% if bank_account.bic %}, {{ bank_account.bic }},{% endif %} to settle the following Thalia Pay payments:
```

Comment:
```suggestion
We would like to inform you that in the upcoming days, we will withdraw an amount of € {{ total_amount|floatformat:2 }} from your bank account {{ bank_account.iban }} {% if bank_account.bic %}, {{ bank_account.bic }},{% endif %} to settle the following Thalia Pay payments:
```

Reply:
But this email needs to be send 1 month in advance doesn't it?

Reply:
14 days*

Reply:
However it is a bit ambiguous as people themselves take the initiative of paying. So you could say that them clicking the 'I want to pay with Thalia Pay' button is already the so-called pre-notification.

Reply:
I think these are the exact details we should discuss with the board